### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by wildsm**
Deleted broken link

### CODE REVIEW

File: doc/README [hunk 1]
```diff
@@ -35,10 +35,6 @@ To get CUTEst working, here is what you need to do:
 
     To obtain problems from the CUTEr/st and related test sets, see
 
-       http://tracsvn.mathappl.polymtl.ca/trac/cuter/wiki/ProblemCollections
-
-    or
-
        http://www.cuter.rl.ac.uk/Problems/mastsif.shtml
```
Comment (dpo) [inline doc/README:38]:
```suggestion
       https://bitbucket.org/optrove/sif/src/master
```
That's where the up-to-date problems now live.

**Review by dpo — CHANGES_REQUESTED** [review]
Many thanks! I'm just suggesting to also update the link for to where the problems actually live.

File: doc/README (hunk 1 above)
Comment (wildsm) [reply doc/README:38]: Even better, works well

**Review by wildsm — COMMENTED** [review]
Updated

**Comment by dpo** [comment]
Thank you, sir!

**Pull request merged**